clarity-ext-scripts/scripts/reportCOVID19samples.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Dmitry Chekmenev, 2020
+# Cron job which extract all samples with their UDFs of Covid-19 Clarity LIMS Project
+
+set -euo pipefail
+
+current_date=`date +%Y%m%d`
```

Comment:
I prefer the `$()` syntax, it's safer to use everywhere
```suggestion
current_date=$(date +%Y%m%d)
```